src/User32.Desktop/User32.cs

```diff
@@ -1434,7 +1454,7 @@ public static partial class User32
         ///         call <see cref="GetLastError" />.
         ///     </para>
         /// </returns>
-        [DllImport(nameof(User32), SetLastError = true)]
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
```

Comment:
Why set this to `CharSet.Unicode` ? Where is any `string` being marshaled? if you mean `strings` inside the `MSG` structure, it should be on the structure and not on the function, I think.